experiments/list_unsupported_build.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+echo >&2 "Project to build: ${project_dir}"
+project_name=$(basename "${project_dir}")
+path="$(cd "$(dirname "${project_dir}")"; pwd)/${project_name}"
+include_path=""
+
+for foldername in $(find ${path} -type d -name "*" | LC_ALL=posix sort ); do
+   count=`ls -1 ${foldername}/*.${spec_ext} 2>/dev/null | wc -l`
+   if [ $count != 0 ]
+   then
+      include_path="${include_path} -I ${foldername}"
+   fi
+done
+
+echo "${project_dir}: Unsupported features\n" > "$project_name".txt
```

Comment:
I think that  _all_ the places in this script where it currently does redirection to a file of some type should just go to stdout/stderr as normal. Again, the end user will not want/be able to alter their build system to send stuff to various separate files so this "fake" build script should do the same - i.e. all output is on either stderr or stdout, and never to a magic file.